src/index.css

```diff
@@ -1,4 +1,4 @@
-body {
+/* body {
   margin: 0;
   font-family:
     -apple-system,
```

Comment:
- [ ] Great work so far 👍 but it would be nice if you remove the commented piece of code from your index.css file because this file only contains commented code 😨 Kindly remove all the commented code from this file to make your codebase clean as suggested by the previous reviewer 😊